Alignment/OfflineValidation/plugins/PrimaryVertexValidation.cc

```diff
@@ -3700,15 +3696,15 @@ void PrimaryVertexValidation::fillDescriptions(edm::ConfigurationDescriptions& d
   // track filtering
   edm::ParameterSetDescription psd0;
   TrackFilterForPVFinding::fillPSetDescription(psd0);
-  psd0.add<int>("numTracksThreshold", 0);  // HI only
+  HITrackFilterForPVFinding::fillPSetDescription(psd0);  // HI only
   desc.add<edm::ParameterSetDescription>("TkFilterParameters", psd0);
 
   // PV Clusterization
   {
     edm::ParameterSetDescription psd0;
     {
       edm::ParameterSetDescription psd1;
-      DAClusterizerInZ_vect::fillPSetDescription(psd1);
+      DAClusterizerInZT_vect::fillPSetDescription(psd1);
```

Comment:
why is this change necessary here? The clusterizer in ZT is never used in this module.

Reply:
@mmusich 
here it is necessary to update the PSet to the one of the ZT clusterizer, which contains all the parameters of both Z and ZT clusterizers, because otherwise the tests of the PVValidation were failing. The error was caused by the missing parameters in the config.

Reply:
This happens because you import here `DA_vectParameters`

https://github.com/cms-sw/cmssw/blob/6b63e89eb5b789dbfe3b19deb72d6d5688e705fb/Alignment/OfflineValidation/test/testPrimaryVertexRelatedValidations_cfg.py#L288

which is defined as:

https://github.com/cms-sw/cmssw/blob/6b63e89eb5b789dbfe3b19deb72d6d5688e705fb/RecoVertex/PrimaryVertexProducer/python/OfflinePrimaryVertices_cfi.py#L7

and is auto-generated by 

https://github.com/cms-sw/cmssw/blob/6b63e89eb5b789dbfe3b19deb72d6d5688e705fb/RecoVertex/PrimaryVertexProducer/plugins/PrimaryVertexProducer.cc#L407

This is assuming that all use-cases of `DA_vectParameters` are actually using the ZT-clusterizer (which is not the case). 
I think it would be clearer to create two `PSet`-s instead: one with the full (Z-T) set of parameters and one with only the ones strictly needed by the clusterizer in z.

Reply:
given https://github.com/cms-sw/cmssw/pull/43592#issuecomment-1910163530, I gather this (admittedly minor) point will be followed up in a subsequent PR.

Reply:
@mmusich @mandrenguyen I have opened https://github.com/cms-sw/cmssw/issues/43787 to track this residual issue